pkg/chartutil/chartfile.go

```diff
@@ -74,7 +74,7 @@ func IsChartDir(dirName string) (bool, error) {
 		return false, err
 	}
 	if chartContent == nil {
-		return false, errors.Errorf("chart metadata (%s) missing", ChartfileName)
+		return false, errors.Errorf("(%s) : No such file or directory", ChartfileName)
```

Comment:
This doesn't seem right. A Chart.yaml file that has no content (e.g. `echo "" > mychart/Chart.yaml`) can trigger this error, which means the error is misleading.

Can you please write a unit test in `chartfile_test.go` that covers the case you are trying to fix?